controller/src/main/java/io/pravega/controller/server/rpc/grpc/GRPCServerConfig.java

```diff
@@ -35,4 +35,39 @@
      * @return The RPC port which has to be registered with the cluster used for external access.
      */
     Optional<Integer> getPublishedRPCPort();
+
+    /**
+     * Fetches the settings which indicates whether authorization is enabled.
+     *
+     * @return Whether this deployment has auth enabled.
+     */
+    boolean isAuthorizationEnabled();
+
+    /**
+     * Fetches the location of passwd file for the users.
+     * This uses UNIX password file syntax. It is used for default implementation of authorization
+     * and authentication.
+     *
+     * @return Location of password file.
+     */
+    String getUserPasswdFile();
```

Comment:
This is taken care of in #2126. The API `void setServerConfig(GRPCServerConfig serverConfig);` in `PravegaAuthHandler.java` handles this. I think it will be easiest to put all the config in `GRPCServerConfig`. Do let me know how that sounds.